src/test/java/org/apache/bcel/verifier/VerifyBadClassesTestCase.java

```diff
@@ -122,6 +123,7 @@ public void testB307() {
      * BCEL-308: NullPointerException in Verifier Pass 3A
      */
     @Test
+    @Disabled("ConstantPool item could be null even when class file is valid")
```

Comment:
Why is this disabled?

Reply:
This test failed after I removed the verification logic, but my new suggestion is enough to pass this case so I reverted this `@Disabled` annotation. 🤝